api/api_ethereum.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"github.com/klaytn/klaytn/datasync/downloader"
+	"github.com/klaytn/klaytn/governance"
+	"github.com/klaytn/klaytn/node/cn/filters"
+)
+
+// EthereumAPI provides an API to access the Klaytn through the `eth` namespace.
+// TODO-Klaytn: Removed unused variable
+type EthereumAPI struct {
+	publicFilterAPI     *filters.PublicFilterAPI
+	governanceKlayAPI   *governance.GovernanceKlayAPI
+	publicDownloaderAPI *downloader.PublicDownloaderAPI
+
+	publicKlayAPI            *PublicKlayAPI
+	publicBlockChainAPI      *PublicBlockChainAPI
+	publicTransactionPoolAPI *PublicTransactionPoolAPI
+	publicAccountAPI         *PublicAccountAPI
+}
```

Comment:
@jimni1222 
Can we handle the `Syncing` APIs using the current design?
There are two `Syncing` APIs, one in `PublicDownloaderAPI` and the other in `PublicKlayAPI`. (Same name but different function signatures).

Because Golang does not support method overloading(it means using same name and different arguments), 
we should separate APIs.

Reply:
thank you for comment. i modified this logic. can you check?